sklearn/metrics/scorer.py

```diff
@@ -430,6 +485,14 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
         For example ``average_precision`` or the area under the roc curve
         can not be computed using discrete predictions alone.
 
+    copy_classes : string or None, default='labels'
+        The name of the scorer parameter into which the classes_ attribute of
+        the estimator will be passed through. If None is passed, then the
+        classes_ atribute will not be copied.
+
+        It should be set only if a user-defined metric function is being used
```

Comment:
pass_classes is probably better.

its not always the training set classes. if the user defined the "classes" argument (though not available for all yet), then it'll not be just training classes. maybe we can just keep it as it is here and give more explanations somewhere else, maybe [here](http://scikit-learn.org/stable/modules/model_evaluation.html#scoring)?